### ISSUE TITLE
TEA => Display Issues in High School Physics

### ISSUE THREAD

**Issue by Stackblocks** [Connexions/cnx-recipes#311]
# Scenario

This issue collects the various display issues encountered in TEA High School Physics. Individual issues can be spun off as necessary.

# Reproducibility Details

- **Impact area**: Gateway Staging
- **User**: openstax@texasgateway.org
- **URL / Location**: stg.texasgateway.org
- **How many times were you able to reproduce the issue?** n/a

### 1. [Ch. 8 Chapter Review](https://stg.texasgateway.org/resource/chapter-review-6?book=256821) -- superscript v. subscript issues
![image](https://user-images.githubusercontent.com/12678235/29420776-9dbefea0-8338-11e7-8317-e86cbba1dea8.png)
Per Helene:
>sub & sup tags styled improperly in the Gateway
>```
>sub {
>    bottom: -0.25em;
>}
>``` 
>needs to be adjusted. Line height might also be an issue.

### 2. [Ch. 9.1 Watch Physics "Work & Energy"](https://stg.texasgateway.org/resource/91-work-power-and-work%E2%80%93energy-theorem?book=256821) -- Line break & hyphen not carried over on "True or False"
![image](https://user-images.githubusercontent.com/12678235/29421370-7e70685c-833a-11e7-901b-88c32dd10c49.png)

### 3. [Book-wide](https://stg.texasgateway.org/book/hs-physics) -- Math display issues (MathJax related?)
![image](https://user-images.githubusercontent.com/12678235/29421710-8b335436-833b-11e7-8e4b-b2bcea91d0dd.png)
Per Helene:
>Display is controlled in CSS on Gateway. `.MathJax` class should be amended to include padding or margins

### 4. [Book-wide](https://stg.texasgateway.org/resource/72-newtons-law-universal-gravitation-and-einsteins-theory-general-relativity?book=316281) -- Section Learning Objectives title shown 3x
![image](https://user-images.githubusercontent.com/12678235/29532084-0b5a46e6-8671-11e7-967f-40ab1b377d9d.png)

# Environment:

- **OS**: macOS
- **Browser**: Chrome

**Comment by helenemccarron** [comment]
- screenshot #1 
the list display is caused by ` display: flex;` on that element in the Gateway's CSS, it needs to be removed

**Comment by helenemccarron** [comment]
- screenshot #3 that character isn't supported.

**Comment by helenemccarron** [comment]
- screenshot 3 - sub & sup tags styled improperly in the Gateway
```
sub {
    bottom: -0.25em;
}
``` 
needs to be adjusted. Line height might also be an issue.

**Comment by helenemccarron** [comment]
- last screenshot, mathjax. Display is controlled in CSS on Gateway.
`.MathJax` class should be amended to include padding or margins

**Comment by philschatz** [comment]
The first item should be resolved with https://github.com/Connexions/cnx-recipes/issues/286#issuecomment-324109223

**Comment by helenemccarron** [comment]
#4 resolved in https://github.com/Connexions/cnx-recipes/pull/297/commits/b1354ca7315da85ea63f79ffd142b6fc5f9f8cc2

**Comment by Stackblocks** [comment]
Tested with Helene's tea-styles.css and updates made by the TEG. Resolved.
